web/src/EducationBenchmarking.Web/Validators/FinancialPlanStages/OtherTeachingPeriodsConfirmStageValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using EducationBenchmarking.Web.Domain.FinancialPlanStages;
+using FluentValidation;
+
+namespace EducationBenchmarking.Web.Validators.FinancialPlanStages
+{
+    public class OtherTeachingPeriodsConfirmStageValidator : AbstractValidator<OtherTeachingPeriodsConfirmStage>
+    {
+        public OtherTeachingPeriodsConfirmStageValidator()
+        {
+            RuleFor(p => p.Proceed)
+                .Must(x => x.HasValue)
+                .WithMessage("Please select yes or no")
```

Comment:
Can we update the message to follow design system guidance?

Select yes if [whatever it is is true]

https://design-system.service.gov.uk/components/radios/#error-messages

Reply:
Updated in latest commit